query1.php

```diff
@@ -60,7 +64,10 @@
 								{	
 								?>
 								<!--php is above. HTML is below. Used to output the query results-->
-								<center><h3><?php echo $output['Total_Time']; ?></h3></center>
+								<center>
+									<img style="width: 100%; max-width: 15px;" src="images/clock.png" alt="Clock Icon">
```

Comment:
#added a clock icon to all playlist pages to support the idea of the total playlist duration